### PR TITLE
Fix Android Studio Warnings

### PR DESCRIPTION
**Pull request by jainv4156**
## Purpose / Description
Fix Android Studio code inspection warnings.
## Fixes
* Fixes #13282

## Approach

Followed the suggestion of android studio 

## How Has This Been Tested?
Ran unit tests locally.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by xenonnn4w** [comment]
rebase these commits , make a single one !

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt [hunk 1]
```diff
@@ -204,7 +204,8 @@ open class DeckPicker :
     private var backButtonPressedToExit = false
     private lateinit var deckPickerContent: RelativeLayout
 
-    @Suppress("Deprecation") // TODO: Encapsulate ProgressDialog within a class to limit the use of deprecated functionality
+    @Suppress("Deprecation")
+    private // TODO: Encapsulate ProgressDialog within a class to limit the use of deprecated functionality
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:208]: This isn't properly formatted, it should be something like:

```
// TODO rest of the text
@Suppress("Deprecation")
private var progressDialog: android.app.ProgressDialog? = null
```

File: AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt [hunk 2]
```diff
@@ -273,7 +273,13 @@ object ReadText {
                         }
                     }
 
-                    @Deprecated("")
+                    @Deprecated(
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt:276]: This is not how to fix this issue, you just need to add 

`@Suppress("DeprecatedCallableAddReplaceWith")`

above the `@Deprecated` annotation to silence it. We don't have any alternative to replace this deprecated method.

**Review by lukstbit — CHANGES_REQUESTED** [review]
Thanks you for contributing. I've left two issues for you to fix and then we can merge this.

**Review by lukstbit — APPROVED** [review]
Thanks for making the changes!
This is fine in terms of made changes, one small nitpick is to change the commit message to something more descriptive of the actual changes like: 

_Fix Android Studio warnings in several files_

so we have a nicer git history.

Also, thanks for rebasing and force pushing, it makes merging easier.

**Comment by jainv4156** [comment]
sure

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
Also awaiting a better commit message

Thanks!

**Pull request merged**